source/propOr.js

```diff
@@ -3,9 +3,8 @@ import pathOr from './pathOr';
 
 
 /**
- * If the given, non-null object has an own property with the specified name,
- * returns the value of that property. Otherwise returns the provided default
- * value.
+ * If the given object has an property with the specified name, returns the value of that
```

Comment:
```suggestion
 * If the given object has a property with the specified name, returns the value of that
```

Replace 'an' with 'a'.